frustratometer/optimization/optimization.py

```diff
@@ -152,9 +153,9 @@ def heterogeneity_approximation(seq_index):
     return het
 
 @numba.njit
-def montecarlo_steps(temperature, model_h, model_J, mask, seq_index, Ep=100, n_steps = 1000, kb = 0.008314) -> np.array:
+def montecarlo_steps(temperature, model_h, model_J, mask, seq_index, Ep=100, n_steps = 1000, kb = 0.008314, excluded='',valid_indices=[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]) -> np.array:
```

Comment:
check if you are using excluded anymore, if not just pass valid_indices. Another way to set a default value is to say `valid_indices = None` and then define it below as `valid_indices=range(len(_AA)) if not valid_indices else valid_indices`